build/http/config.xml

```diff
@@ -244,12 +244,13 @@
         </wan>
         <lan>
             <enable>1</enable>
-            <if>em0</if>
-            <ipaddr>10.2.0.1</ipaddr>
+            <if>em1</if>
+            <ipaddr>dhcp</ipaddr>
+            <subnet/>
             <!--
             <ipaddr>192.168.1.1</ipaddr>
-            -->
             <subnet>24</subnet>
+            -->
             <ipaddrv6>track6</ipaddrv6>
             <subnetv6>64</subnetv6>
             <media/>
```

Comment:
Why are you removing the entire LAN segment configuration in the first place? Using hosts DHCP here for a network we want to be managed by OPNsense makes no sense to me